apps/framework-cli/src/infrastructure/processes/kafka_clickhouse_sync.rs

```diff
@@ -464,7 +464,7 @@ async fn sync_kafka_to_clickhouse(
             select! {
                 received = subscriber.recv() => match received {
                     Err(e) => {
-                        debug!("Error receiving message from {}: {}", source_topic_name, e);
+                        warn!("Error receiving message from {}: {}", source_topic_name, e);
```

Comment:
Why is this a warning?

Reply:
My hunch is that the consumer did not resume after the `PollExceeded` because some internal state is messed up. Better to make sure this error get logged.